src/main/java/uk/gov/hmcts/probate/controller/ExceptionRecordController.java

```diff
@@ -92,14 +96,17 @@
             switch (formType) {
                 case PA8A:
                     callbackResponse = erService.createCaveatCaseFromExceptionRecord(erRequest, warnings);
+                    logTransformationCallback(callbackResponse);
```

Comment:
Are these logs to be disabled now or after UAT?

Reply:
I did remove them, but decided to take them from demo - ill remove them again. They should not be in master anyway